src/css/override_components.scss

```diff
@@ -45,6 +48,22 @@
   }
 }
 
+// kludge for targeting dashboard
+.js-app {
```

Comment:
Same as above, maybe `disclaimer-lg`, or wrap the disclaimer in the `header-lg` or whatever class.